lib/commands/element.js

```diff
@@ -36,5 +36,31 @@ commands.touchMove = async function (elementId, x, y) {
   return await this.uiautomator2.jwproxy.command(`/touch/move`,'POST', {params});
 };
 
-Object.assign(extensions, commands);
+
+helpers.setElementValue = async function (keys, elementId, replace = false) {
+  let text = keys;
+  if (keys instanceof Array) {
+    text = keys.join("");
+  }
+
+  let params = {
+    "id" : elementId,
```

Comment:
As [SendKeysToElement ](https://github.com/appium/appium-uiautomator2-server/blob/master/app/src/main/java/io/appium/uiautomator2/handler/SendKeysToElement.java#L32) handler in uiautomator2-server module expects `id `in json body, just to be in sync with handler customized json body